redesign-project/src/index.html

```diff
@@ -0,0 +1,67 @@
+    alt=" " class="background">
+
+    <img src="https://lh3.googleusercontent.com/BrkYF7ymKQaMQHOo8chYsPzUVzu1hHymNJpLfrXHKnL0y--NFu39TYjBucjWHnnFIkuA-6Htc4mEoSTrqPa48_B7t84GEs8mZjXzAzjPizGa9HK2A-0R7An6_zJDDkdxMGJV9yYXz1W6T4Ndd38JkqlPegUuF9O1D8p4YIxyU97MRrZIckkP70BOp6aejrJHDK1P1qKZFPQGYxZiaFJbRLPGf0bfGXA9Qj--YY4PyvtrT4k3bJcIXR0MyRUtARNYoUEmgISBeTcr6JUh_HqBj-agO-g_K3oyMM1kJad-e1SKtJPIfE4UiAofbPXSFtdg7F2l-tNX6MW0BmL5O_UtZtQwbKLV-Pw9tDvTqNTxbEpAQQ8qRVt-yEssZx44kx4eYVkieu_p7iCpBdSYGAOCx6kUnfbHh93yYDDY-0QhsSTg0XGf69jnjezV0ke8m7XwN0Ru7sKacRocLfftllJauZUXu1U7bUqAqUj_tHATfeEtRYSvR8flTQdAwCTA-iR7zdQftFtboEwIU-mzfNVXCzHbVXWLby0qYMhE2lqT2IXwjvpUQ7k_YktBd_gZKgSQNq2_U9bPkLCTiu2cM5BDxBxccMO6TGvFkxAIyX5n1cgb-z-jjYD4YOAXggSWgVTwCklztIfbG0htXGeWjqPeb43laoFZ0EncC4jUNKE-qRpK5Q9LVfAGy9dwibcds70LDS_MtRL4JSrra6GlmA=w2492-h1304-no"
+    alt=" " class="desktop_background">
+
+    <div class="rectangle"></div>
+
+    <div class="rectangle_large"></div>
+
+    <img src="https://static1.squarespace.com/static/59cee597bebafbd725f0d1f8/t/59cee7cad2b85766102d70fa/1540697069191/?format=1500w"
+        alt="Emma's Tea Spot: A Proper British Experience" class="header">
+
+
+    <ul>
+        <li>menu</li>
+        <li>reservations</li>
+        <li>events</li>
+        <li>shop</li>
+        <li>subscribe</li>
+    </ul>
+
+ <h1>Upcoming Events</h1>
+
+    <ul>
+        <li>Our One-Year Anniversary Party!<br>Nov 24, 2018<br>8:00 AM – 4:00 PM</li>
+        <li>Kids Cook<br>Dec 1, 2018<br>9:00 AM – 10:00 AM</li>
+        <li>First Friday<br>Dec 7, 2018<br>6:30 PM – 8:30 PM</li>
+        <li>Kids' Tea Party<br>Dec 27, 2018<br>5:30 PM – 6:30 PM</li>
+        <li>First Friday<br>Jan 4, 2019<br>6:30 PM – 8:30 PM</li>
+    </ul>
+
+ <h2>Hours<h2>
+
+<ul>
+    <li>Monday: 10:30-6:30</li>
+    <li>Tuesday: Closed</li>
+    <li>Wednesday: 10:30-6:30</li>
+    <li>Thursday: 10:30-6:30</li>
+    <li>Friday: 10:30-8:00</li>
+    <li>Saturday: 9-4</li>
+    <li>Sunday: 10-3</li>
+</ul>
+
+<h3>Contact</h3>
+
+<ul>
+    <li>410 . 444 . 1718</li>
+    <li>5500 Harford Road<br>Hamilton, MD<br>21214</li>
+</ul>
+
+    <img src="https://lh3.googleusercontent.com/iXmo4mpXjCyGuFgO6wE8O9asLT2oBT5hrYqjcuoeXSRE_ya_pGN2nyqfevKyPP_sOdJuRO_UOclvtrAF03LRdv0kE4zsANxlTMhE54M-0qtyMlysVJY2vb6h1PS0FjlNdmiJ7mRr5pdhXQ1FwOdqzklCZC9fTfl9E5UVUGcwUPnuUmmwYSXQwpd-_YKo-xrvmb_Q0aJwbQVVx4DTx54k3kmFRPoFxGe_yXhoUgLe4JCuCwd5RHyGbJkUTiuQ3JpuZBklbRPfgit1rE-yGzPFijnKHK1dTp4LB4fB5fh3GIqw0hUBPAsenJkGzQRi7enFqxBDU5LkAoFFI1uIdz2zX-gyU6jO683o754j_pZxLRufcR1XiBxvVzRqv31UFag9B3ajviRhbVUbQX-bU0sUj_Q49OSmrWaPoO221vanAJAD_KzSyihw-pniCr0Gye39F6VuMXT9aIdfRfh0Xom6KbuiTB0KFolb0VnXBCeOBHwiyjuxQET-dShWovaNYA20MrRyoWW4j27N4EriIayxiJLUxl7sLAXaGWdJgh92vXVgdRXUCZMOlYQ6W9QkNwOtdkAH9DB9BVO2GTiXKaypC17nnNqwMjbNFNdne6tS5HsXXNylxiZTyRbdITcgf91ECELcw0nZTjncL2zavR02T3s=s128-no"
+        alt="Social media icon for Instagram." class="footer footer__insta">
```

Comment:
The `alt` text here can be simplified to "Instagram logo" and the same for the others.